js/defaults_dialog.js

```diff
@@ -729,14 +883,14 @@ helper.defaultsDialog = (function () {
 
     privateScope.setSettings = function (selectedDefaultPreset) {
         //Save analytics
+        var currentProfile = parseInt($("#profilechange").val());
         googleAnalytics.sendEvent('Setting', 'Defaults', selectedDefaultPreset.title);
         Promise.mapSeries(selectedDefaultPreset.settings, function (input, ii) {
             return mspHelper.getSetting(input.key);
         }).then(function () {
             Promise.mapSeries(selectedDefaultPreset.settings, function (input, ii) {
-                return mspHelper.setSetting(input.key, input.value);
+                return privateScope.setSetting(input.key, input.value);
```

Comment:
It shopuld use `return mspHelper.setSetting(input.key, input.value);` not this internally invented method